src/components/Reports/ReportLayout/NavReportsList/NavReportsList.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {
+  Box,
+  BoxProps,
+  List,
+  styled,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import NavReportListItem, { ReportOption } from './NavReportListItem';
+
+const reportItems: ReportOption[] = [
```

Comment:
Could we share this with the nav drop-down somehow? That can happen in another PR.

Reply:
https://github.com/CruGlobal/mpdx-react/pull/103